scootapi/client/smoke_test_cmd.go

```diff
@@ -113,8 +136,11 @@ func (c *Client) generateAndRunJob(timeout time.Duration) error {
 				return fmt.Errorf("Error getting status: %v", err.Error())
 			}
 		}
-
-		fmt.Println("Status Update Job", jobId.ID, status.Status)
+		// get current version of jobStatusMap
+		jobStatusMap := <-ch
```

Comment:
So you're handing it back and forth a whole bunch. It looks like this works; it might work even better to just send the name and status back to one goroutine that both owns the maps and prints the status.